App/Lettuce/Sources/Lettuce/UI/MessagesView/Subviews/LastMessageView.swift

```diff
@@ -4,17 +4,23 @@ import SwiftUI
 struct LastMessageView: View {
 
     let message: Message
+    let read: Bool
 
     private let fontColor = (Color(red: 0.24, green: 0.24, blue: 0.26).opacity(0.6))
+    private let readFontColor = (Color(red: 0.24, green: 0.24, blue: 0.26).opacity(1.0))
```

Comment:
Please check `DesignSystem.Color`. You can add a custom type (if needed) for the message colors but I think we should move as much as possible to `DesignSystem`.

Reply:
fixed